tensilelite/Tensile/KernelWriter.py

```diff
@@ -1587,13 +1587,17 @@ def noLoadLoopBody( self, kernel, tensorParametersA, tensorParametersB, pack, is
       isLastLoop = not isNGLL
       if u == 0:
         if not isLastLoop:
-          self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersA)  # local write in loopcnt N targets data for loopcnt N+1
-          self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersB)
+          if dsWriteBA==True:
+            self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersB, swapAB=1)
+            self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersA, swapAB=1)
+          else:
+            self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersA)  # local write in loopcnt N targets data for loopcnt N+1
+            self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersB)
```

Comment:
In the current scheduling, we always schedule lwa first then lwb second. 
Put B in lwa code can easily change the order.